extensions/amp-form/0.1/amp-form.js

```diff
@@ -279,6 +291,33 @@ export class AmpForm {
   }
 
   /**
+   * Handles response redirect throught the AMP-Redirect-To response header.
+   * @param {../../../src/service/xhr-impl.FetchResponse} response
+   * @private
+   */
+  maybeHandleRedirect_(response) {
+    // If the `AMP-Redirect-To` header is set. Redirect the user to the
+    // given URL.
+    const redirectTo = /** @type {string} */ (response.headers &&
```

Comment:
Ick. Can we break this up into two tests to avoid the type coercion?

```js
if (!response.headers) {
  return;
}
const redirectTo = response.headers.get(HEADER);